index.js

```diff
@@ -1,11 +1,25 @@
-import { config } from './config.js';
 import express from 'express';
 
-const app = express();
+export default class Server {
+    constructor (port, app) {
```

Comment:
Let's incupsulate `app` in our server. 
I mean we can call `express()` inside our class constructor instead of passing it as an argument.